### PR TITLE
update tutorial.md add tips section

### PR DESCRIPTION
**Pull request by isaac-tb1010**
I would like to share details on how Spotify returns its data and how to access information with Python - a tips section if you will. I added a small tutorial/code block to assist. I would also like to add a walkthrough for the code flow sections since the code isn’t explained.

### CODE REVIEW

**Review by spire-carlos — APPROVED** [review]

File: TUTORIAL.md [hunk 1]
```diff
@@ -78,4 +78,140 @@ D. Close main.py and return to the directory that contains main.py. You can then
...
+                                              'SK',
+                                              'SV',
+                                              'TH',
+                                              'TN',
+                                              'TR',
+                                              'TW',
+                                              'US',
+                                              'UY',
+                                              'VN',
+                                              'ZA'],
```
Comment (dieser-niko) [inline TUTORIAL.md:170]: Please minimize this.

File: TUTORIAL.md [hunk 2]
```diff
@@ -78,4 +78,140 @@ D. Close main.py and return to the directory that contains main.py. You can then
 
 E. You may see a window open in your browser asking you to authorize the application. Do so - you will only have to do this once.
 
-F. Return to your terminal - you should see all of Taylor's albums printed out there.
\ No newline at end of file
+F. Return to your terminal - you should see all of Taylor's albums printed out there.
+
+## Accessing specific data
+The data returned is an object of type dictionary. Accessing the data is quite easy and then can be transformed into json and string format. Below is an example of what is returned from querying the API for a user's top tracks.
```
Comment (dieser-niko) [inline TUTORIAL.md:84]: > can be transformed into json and string format.

The wording could be improved. It reads like the user has to transform the json output themselves.

```suggestion
The data returned is an object of type dictionary. Accessing the data is quite easy as the library already parses the JSON response. Below is an example of what is returned from querying the API for a user's top tracks.
```

File: TUTORIAL.md [hunk 3]
```diff
@@ -78,4 +78,140 @@ D. Close main.py and return to the directory that contains main.py. You can then
...
+```
```
Comment (dieser-niko) [inline TUTORIAL.md:85]: Add language to code block

```suggestion
```python
```

File: TUTORIAL.md [hunk 4]
```diff
@@ -78,4 +78,140 @@ D. Close main.py and return to the directory that contains main.py. You can then
...
+                     'type': 'track',
+                     'uri': 'spotify:track:5r6fLJUE1yMtEZTe71oDyy'}],
+    'limit': 20,
+    'next': 'https://api.spotify.com/v1/me/top/tracks?limit=20&offset=20&time_range=short_term',
+    'offset': 0,
+    'previous': None,
+    'total': 50}
+```
+To provide a clear way of access and reformatting we can write some code to grab some important information that we see fit for our use case:
+```
```
Comment (dieser-niko) [inline TUTORIAL.md:192]:
```suggestion
```python
```

File: TUTORIAL.md [hunk 5]
```diff
@@ -78,4 +78,140 @@ D. Close main.py and return to the directory that contains main.py. You can then
...
+            "song_name": item["name"],
+            "artist_name": item["artists"][0]["name"],
+            "url": item["artists"][0]["external_urls"],
+            "uri": item["uri"],
+        }
+        top_track_array.append(top_tracks_ids)
+    return results
+```
+This transforms the raw response into something useful for our/your program:
+```
```
Comment (dieser-niko) [inline TUTORIAL.md:209]:
```suggestion
```python
```

**Review by dieser-niko — CHANGES_REQUESTED** [review]
I've added some suggestions that would improve the document a bit.

**Comment by stephanebruckert** [comment]
Closing as we already have enough examples in https://github.com/spotipy-dev/spotipy/tree/master/examples

**Pull request closed without merging**